vc-defer.el

```diff
@@ -309,10 +309,11 @@ global auto reverts back on, use \\[global-auto-revert-mode]."
     ;; are very intrusive in the trace output.
     (global-auto-revert-mode 0)
     (dolist (buffer (buffer-list))
-      (if auto-revert-mode
```

Comment:
I'd just remove these if statements. (auto-revert-mode -1) should do the right thing in all cases.

Reply:
Thanks for suggestion. I have updated https://github.com/google/vc-defer/pull/4/commits/d1388b9c7c86d7b01a967ab7ace0106f1734d5c8